### ISSUE TITLE
Removal from npmjs.com

### ISSUE THREAD

**Issue by sourcx** [sethmcl/jquery-ui#1]
Hi,

Can you remove this outdated version from npmjs.com please? 
The naming is confusing and I accidentally included your old jqueryui in my project instead of jquery-ui.

Cheers!

**Comment by ctownshend** [comment]
Hey, happened to me too. Can you remove this from npmjs?

**Comment by thusitha-cs** [comment]
Yup. This package is so outdated now, and making issues where we have used it.
